feat(wallet)!: support for calculating/setting fees for non eip1559 compliant transactions

saledjenic opened the pull request:
What's changed:
- Introduced `EIP1559Enabled` field in the Network struct, which will be set forthe  chain in the path response
- Introduced `GasPrice` field in `PathTxCustomParams` that will be used for setting custom fees for non-EIP-1559 chains
- Updated Router methods to handle suggested fees and gas prices based on EIP-1559 flag
- Refactored fee estimation logic to accommodate both EIP-1559 and non-EIP-1559 fee structures
- Improved path handling to ensure correct fee application for transactions and approvals

Clients need to:
- check `EIP1559Enabled` field of the `FromChain` field for each path from the Best router that is being processed
- based on that the client needs to either use values from `SuggestedNonEIP1559Fees` or `SuggestedLevelsForMaxFeesPerGas` when preparing estimated time or suggested fee values
- `ApprovalGasPrice` and `TxGasPrice` are added to the `Path` and they represent the value that will be used for the tx execution
- Custom gas price can be set via `GasPrice` field of `PathTxCustomParams` via `SetCustomTxDetails` API endpoint
- API endpoint `GetTransactionEstimatedTimeV2` extended with `gasPrice` parameter, now it's able to estimate time for EIP1559 compliant and non-compliant chains

saledjenic committed:
feat(wallet)!: support for calculating/setting fees for non eip1559 compliant transactions

- Introduced `EIP1559Enabled` field in the Network struct, that will be set for chain in the path response
- Introduced `GasPrice` field in `PathTxCustomParams` that will be used for setting custom fees for
non-EIP-1559 chains via `SetCustomTxDetails` API endpoint
- API endpoint `GetTransactionEstimatedTimeV2` extended with `gasPrice` parameter
- Updated Router methods to handle suggested fees and gas prices based on EIP-1559 flag
- Refactored fee estimation logic to accommodate both EIP-1559 and non-EIP-1559 fee structures
- Improved path handling to ensure correct fee application for transactions and approvals

status-im-auto commented:
### Jenkins Builds
<details>
<summary>Click to see older builds (104)</summary>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :x: | f2672feb | [#1](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6582/1/) | 2025-05-15 12:35:17 | ~2 min | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6582/1/consoleText) |
| :x: | f2672feb | [#1](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6582/1/) | 2025-05-15 12:35:26 | ~2 min | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6582/1/consoleText) |
| :x: | f2672feb | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6582/1/) | 2025-05-15 12:35:47 | ~2 min | `linux` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6582/1/consoleText) |
| :x: | f2672feb | [#1](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6582/1/) | 2025-05-15 12:35:58 | ~2 min | `macos` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6582/1/consoleText) |
| :x: | f2672feb | [#1](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6582/1/) | 2025-05-15 12:36:29 | ~3 min | `windows` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6582/1/consoleText) |
| :heavy_multiplication_x: | f2672feb | [#1](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6582/1/) | 2025-05-15 12:36:29 | ~3 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6582/1//consoleText) |
| :x: | f2672feb | [#1](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6582/1/) | 2025-05-15 12:36:57 | ~3 min | `macos` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6582/1/consoleText) |
| :heavy_multiplication_x: | f2672feb | [#1](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6582/1/) | 2025-05-15 12:38:18 | ~5 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6582/1//consoleText) |
| :x: | f2672feb | [#2](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6582/2/) | 2025-05-15 12:48:25 | ~1 min | `windows` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6582/2/consoleText) |
| :x: | f2672feb | [#2](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6582/2/) | 2025-05-15 12:48:26 | ~1 min | `macos` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6582/2/consoleText) |
| :x: | f2672feb | [#2](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6582/2/) | 2025-05-15 12:48:46 | ~2 min | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6582/2/consoleText) |
| :x: | f2672feb | [#2](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6582/2/) | 2025-05-15 12:48:52 | ~2 min | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6582/2/consoleText) |
| :heavy_multiplication_x: | f2672feb | [#2](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6582/2/) | 2025-05-15 12:48:52 | ~2 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6582/2//consoleText) |
| :x: | f2672feb | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6582/2/) | 2025-05-15 12:49:18 | ~2 min | `linux` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6582/2/consoleText) |
| :x: | f2672feb | [#2](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6582/2/) | 2025-05-15 12:50:10 | ~3 min | `macos` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6582/2/consoleText) |
| :heavy_multiplication_x: | f2672feb | [#2](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6582/2/) | 2025-05-15 12:51:18 | ~4 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6582/2//consoleText) |
| | | | | | | |
| :x: | 9500b451 | [#3](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6582/3/) | 2025-05-15 12:50:16 | ~1 min | `windows` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6582/3/consoleText) |
| :x: | 9500b451 | [#3](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6582/3/) | 2025-05-15 12:50:24 | ~1 min | `macos` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6582/3/consoleText) |
| :heavy_multiplication_x: | 9500b451 | [#3](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6582/3/) | 2025-05-15 12:51:12 | ~2 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6582/3//consoleText) |
| :x: | 9500b451 | [#3](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6582/3/) | 2025-05-15 12:51:20 | ~2 min | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6582/3/consoleText) |
| :x: | 9500b451 | [#3](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6582/3/) | 2025-05-15 12:51:41 | ~2 min | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6582/3/consoleText) |
| :x: | 9500b451 | [#3](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6582/3/) | 2025-05-15 12:52:06 | ~2 min | `linux` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6582/3/consoleText) |
| :x: | 9500b451 | [#3](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6582/3/) | 2025-05-15 12:53:45 | ~3 min | `macos` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6582/3/consoleText) |
| :heavy_multiplication_x: | 9500b451 | [#3](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6582/3/) | 2025-05-15 12:56:50 | ~4 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6582/3//consoleText) |
| | | | | | | |
| :x: | 033f55c2 | [#4](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6582/4/) | 2025-05-15 13:05:33 | ~1 min | `windows` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6582/4/consoleText) |
| :x: | 033f55c2 | [#4](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6582/4/) | 2025-05-15 13:05:49 | ~2 min | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6582/4/consoleText) |
| :heavy_multiplication_x: | 033f55c2 | [#4](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6582/4/) | 2025-05-15 13:05:50 | ~2 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6582/4//consoleText) |
| :x: | 033f55c2 | [#4](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6582/4/) | 2025-05-15 13:05:51 | ~2 min | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6582/4/consoleText) |
| :x: | 033f55c2 | [#4](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6582/4/) | 2025-05-15 13:06:06 | ~2 min | `macos` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6582/4/consoleText) |
| :x: | 033f55c2 | [#4](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6582/4/) | 2025-05-15 13:06:45 | ~3 min | `linux` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6582/4/consoleText) |
| :x: | 033f55c2 | [#4](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6582/4/) | 2025-05-15 13:07:10 | ~3 min | `macos` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6582/4/consoleText) |
| :heavy_multiplication_x: | 033f55c2 | [#4](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6582/4/) | 2025-05-15 13:07:25 | ~3 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6582/4//consoleText) |
| | | | | | | |
| :heavy_check_mark: | a84a0a29 | [#5](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6582/5/) | 2025-05-15 13:48:30 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250515-134535-a84a0a-pr6582.zip) |
| :heavy_check_mark: | a84a0a29 | [#5](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6582/5/) | 2025-05-15 13:48:59 | ~3 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250515-134546-a84a0a-pr6582.zip) |
| :heavy_check_mark: | a84a0a29 | [#5](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6582/5/) | 2025-05-15 13:49:23 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250515-134539-a84a0a-pr6582.zip) |
| :heavy_check_mark: | a84a0a29 | [#5](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6582/5/) | 2025-05-15 13:49:48 | ~4 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250515-134534-a84a0a-pr6582.aar) |
| :heavy_multiplication_x: | a84a0a29 | [#5](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6582/5/) | 2025-05-15 13:50:22 | ~4 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6582/5//consoleText) |
| :heavy_check_mark: | a84a0a29 | [#5](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6582/5/) | 2025-05-15 13:50:50 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250515-134542-a84a0a-pr6582.zip) |
| :heavy_check_mark: | a84a0a29 | [#5](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6582/5/) | 2025-05-15 13:51:05 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250515-134537-a84a0a-pr6582.zip) |
| :heavy_check_mark: | a84a0a29 | [#5](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6582/5/) | 2025-05-15 13:56:03 | ~10 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6582/5//consoleText) |
| | | | | | | |
| :heavy_check_mark: | a83c44a4 | [#6](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6582/6/) | 2025-05-16 08:55:57 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250516-085302-a83c44-pr6582.aar) |
| :heavy_check_mark: | a83c44a4 | [#6](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6582/6/) | 2025-05-16 08:56:09 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250516-085304-a83c44-pr6582.zip) |
| :heavy_check_mark: | a83c44a4 | [#6](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6582/6/) | 2025-05-16 08:56:37 | ~3 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250516-085312-a83c44-pr6582.zip) |
| :heavy_check_mark: | a83c44a4 | [#6](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6582/6/) | 2025-05-16 08:57:42 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250516-085307-a83c44-pr6582.zip) |
| :heavy_check_mark: | a83c44a4 | [#6](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6582/6/) | 2025-05-16 08:58:16 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250516-085309-a83c44-pr6582.zip) |
| :heavy_check_mark: | a83c44a4 | [#6](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6582/6/) | 2025-05-16 09:00:22 | ~7 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250516-085305-a83c44-pr6582.zip) |
| :heavy_check_mark: | a83c44a4 | [#6](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6582/6/) | 2025-05-16 09:03:07 | ~9 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6582/6//consoleText) |
| :heavy_multiplication_x: | a83c44a4 | [#6](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6582/6/) | 2025-05-16 09:29:22 | ~36 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6582/6//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 5fb4954c | [#7](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6582/7/) | 2025-05-16 09:00:29 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250516-085733-5fb495-pr6582.aar) |
| :heavy_check_mark: | 5fb4954c | [#7](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6582/7/) | 2025-05-16 09:00:35 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250516-085735-5fb495-pr6582.zip) |
| :heavy_check_mark: | 5fb4954c | [#7](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6582/7/) | 2025-05-16 09:00:49 | ~3 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250516-085746-5fb495-pr6582.zip) |
| :heavy_check_mark: | 5fb4954c | [#7](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6582/7/) | 2025-05-16 09:02:17 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250516-085747-5fb495-pr6582.zip) |
| :heavy_check_mark: | 5fb4954c | [#7](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6582/7/) | 2025-05-16 09:03:26 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250516-085821-5fb495-pr6582.zip) |
| :heavy_check_mark: | 5fb4954c | [#7](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6582/7/) | 2025-05-16 09:05:34 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250516-090026-5fb495-pr6582.zip) |
| :heavy_check_mark: | 5fb4954c | [#7](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6582/7/) | 2025-05-16 09:12:43 | ~9 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6582/7//consoleText) |
| :heavy_multiplication_x: | 5fb4954c | [#7](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6582/7/) | 2025-05-16 10:04:54 | ~35 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6582/7//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 8cf8f491 | [#8](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6582/8/) | 2025-05-16 15:30:25 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250516-152727-8cf8f4-pr6582.aar) |
| :heavy_check_mark: | 8cf8f491 | [#8](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6582/8/) | 2025-05-16 15:30:25 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250516-152729-8cf8f4-pr6582.zip) |
| :heavy_check_mark: | 8cf8f491 | [#8](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6582/8/) | 2025-05-16 15:30:58 | ~3 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250516-152738-8cf8f4-pr6582.zip) |
| :heavy_check_mark: | 8cf8f491 | [#8](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6582/8/) | 2025-05-16 15:31:07 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250516-152733-8cf8f4-pr6582.zip) |
| :heavy_check_mark: | 8cf8f491 | [#8](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6582/8/) | 2025-05-16 15:32:53 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250516-152734-8cf8f4-pr6582.zip) |
| :heavy_check_mark: | 8cf8f491 | [#8](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6582/8/) | 2025-05-16 15:33:24 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250516-152730-8cf8f4-pr6582.zip) |
| :heavy_check_mark: | 8cf8f491 | [#8](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6582/8/) | 2025-05-16 15:37:34 | ~9 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6582/8//consoleText) |
| :heavy_multiplication_x: | 8cf8f491 | [#8](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6582/8/) | 2025-05-16 16:01:05 | ~33 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6582/8//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 1f0c405e | [#9](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6582/9/) | 2025-05-19 12:09:34 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250519-120630-1f0c40-pr6582.aar) |
| :heavy_multiplication_x: | 1f0c405e | [#9](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6582/9/) | 2025-05-19 12:09:51 | ~3 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6582/9//consoleText) |
| :heavy_check_mark: | 1f0c405e | [#9](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6582/9/) | 2025-05-19 12:10:00 | ~3 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250519-120640-1f0c40-pr6582.zip) |
| :heavy_check_mark: | 1f0c405e | [#9](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6582/9/) | 2025-05-19 12:10:01 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250519-120630-1f0c40-pr6582.zip) |
| :heavy_check_mark: | 1f0c405e | [#9](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6582/9/) | 2025-05-19 12:11:46 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250519-120637-1f0c40-pr6582.zip) |
| :heavy_check_mark: | 1f0c405e | [#9](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6582/9/) | 2025-05-19 12:11:52 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250519-120634-1f0c40-pr6582.zip) |
| :heavy_check_mark: | 1f0c405e | [#9](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6582/9/) | 2025-05-19 12:13:01 | ~6 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250519-120633-1f0c40-pr6582.zip) |
| :heavy_check_mark: | 1f0c405e | [#9](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6582/9/) | 2025-05-19 12:16:04 | ~9 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6582/9//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 078e0769 | [#10](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6582/10/) | 2025-05-20 06:08:50 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250520-060547-078e07-pr6582.aar) |
| :heavy_check_mark: | 078e0769 | [#10](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6582/10/) | 2025-05-20 06:08:57 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250520-060548-078e07-pr6582.zip) |
| :heavy_check_mark: | 078e0769 | [#10](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6582/10/) | 2025-05-20 06:09:57 | ~4 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250520-060556-078e07-pr6582.zip) |
| :heavy_check_mark: | 078e0769 | [#10](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6582/10/) | 2025-05-20 06:10:24 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250520-060552-078e07-pr6582.zip) |
| :heavy_check_mark: | 078e0769 | [#10](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6582/10/) | 2025-05-20 06:11:02 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250520-060552-078e07-pr6582.zip) |
| :heavy_check_mark: | 078e0769 | [#10](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6582/10/) | 2025-05-20 06:11:37 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250520-060549-078e07-pr6582.zip) |
| :heavy_multiplication_x: | 078e0769 | [#10](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6582/10/) | 2025-05-20 06:16:02 | ~10 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6582/10//consoleText) |
| :heavy_multiplication_x: | 078e0769 | [#10](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6582/10/) | 2025-05-20 06:41:49 | ~35 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6582/10//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 37c47cf9 | [#11](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6582/11/) | 2025-05-20 11:37:29 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250520-113434-37c47c-pr6582.aar) |
| :heavy_check_mark: | 37c47cf9 | [#11](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6582/11/) | 2025-05-20 11:37:38 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250520-113436-37c47c-pr6582.zip) |
| :heavy_check_mark: | 37c47cf9 | [#11](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6582/11/) | 2025-05-20 11:37:49 | ~3 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250520-113444-37c47c-pr6582.zip) |
| :heavy_check_mark: | 37c47cf9 | [#11](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6582/11/) | 2025-05-20 11:38:08 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250520-113438-37c47c-pr6582.zip) |
| :heavy_check_mark: | 37c47cf9 | [#11](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6582/11/) | 2025-05-20 11:39:47 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250520-113439-37c47c-pr6582.zip) |
| :heavy_check_mark: | 37c47cf9 | [#11](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6582/11/) | 2025-05-20 11:40:18 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250520-113436-37c47c-pr6582.zip) |
| :heavy_check_mark: | 37c47cf9 | [#11](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6582/11/) | 2025-05-20 11:44:25 | ~9 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6582/11//consoleText) |
| :heavy_multiplication_x: | 37c47cf9 | [#11](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6582/11/) | 2025-05-20 12:08:52 | ~34 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6582/11//consoleText) |
| | | | | | | |
| :x: | 67535c80 | [#12](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6582/12/) | 2025-05-21 05:40:09 | ~2 min | `macos` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6582/12/consoleText) |
| :x: | 67535c80 | [#12](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6582/12/) | 2025-05-21 05:40:19 | ~2 min | `windows` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6582/12/consoleText) |
| :x: | 67535c80 | [#12](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6582/12/) | 2025-05-21 05:40:21 | ~2 min | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6582/12/consoleText) |
| :x: | 67535c80 | [#12](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6582/12/) | 2025-05-21 05:40:23 | ~2 min | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6582/12/consoleText) |
| :heavy_multiplication_x: | 67535c80 | [#12](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6582/12/) | 2025-05-21 05:40:29 | ~2 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6582/12//consoleText) |
| :x: | 67535c80 | [#12](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6582/12/) | 2025-05-21 05:40:56 | ~2 min | `linux` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6582/12/consoleText) |
| :x: | 67535c80 | [#12](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6582/12/) | 2025-05-21 05:41:39 | ~3 min | `macos` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6582/12/consoleText) |
| :heavy_multiplication_x: | 67535c80 | [#12](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6582/12/) | 2025-05-21 05:42:14 | ~4 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6582/12//consoleText) |
| | | | | | | |
| :heavy_check_mark: | e36f968e | [#13](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6582/13/) | 2025-05-21 06:24:32 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250521-062137-e36f96-pr6582.aar) |
| :heavy_check_mark: | e36f968e | [#13](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6582/13/) | 2025-05-21 06:24:43 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250521-062138-e36f96-pr6582.zip) |
| :heavy_check_mark: | e36f968e | [#13](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6582/13/) | 2025-05-21 06:25:05 | ~3 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250521-062146-e36f96-pr6582.zip) |
| :heavy_check_mark: | e36f968e | [#13](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6582/13/) | 2025-05-21 06:25:13 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250521-062141-e36f96-pr6582.zip) |
| :heavy_check_mark: | e36f968e | [#13](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6582/13/) | 2025-05-21 06:26:49 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250521-062142-e36f96-pr6582.zip) |
| :heavy_check_mark: | e36f968e | [#13](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6582/13/) | 2025-05-21 06:27:27 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250521-062140-e36f96-pr6582.zip) |
| :heavy_multiplication_x: | e36f968e | [#13](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6582/13/) | 2025-05-21 06:31:19 | ~9 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6582/13//consoleText) |
| :heavy_multiplication_x: | e36f968e | [#13](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6582/13/) | 2025-05-21 06:55:48 | ~34 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6582/13//consoleText) |
</details>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | e95ebc3a | [#14](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6582/14/) | 2025-05-21 07:27:09 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250521-072413-e95ebc-pr6582.aar) |
| :heavy_check_mark: | e95ebc3a | [#14](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6582/14/) | 2025-05-21 07:27:24 | ~2 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250521-072424-e95ebc-pr6582.zip) |
| :heavy_check_mark: | e95ebc3a | [#14](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6582/14/) | 2025-05-21 07:28:23 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250521-072414-e95ebc-pr6582.zip) |
| :heavy_check_mark: | e95ebc3a | [#14](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6582/14/) | 2025-05-21 07:29:28 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250521-072420-e95ebc-pr6582.zip) |
| :heavy_check_mark: | e95ebc3a | [#14](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6582/14/) | 2025-05-21 07:31:28 | ~7 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250521-072416-e95ebc-pr6582.zip) |
| :heavy_check_mark: | e95ebc3a | [#14](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6582/14/) | 2025-05-21 07:31:59 | ~7 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250521-072418-e95ebc-pr6582.zip) |
| :heavy_check_mark: | e95ebc3a | [#14](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6582/14/) | 2025-05-21 07:34:27 | ~10 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6582/14//consoleText) |
| :heavy_multiplication_x: | e95ebc3a | [#14](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6582/14/) | 2025-05-21 07:58:56 | ~34 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6582/14//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 77f1a2e2 | [#15](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6582/15/) | 2025-05-21 07:34:51 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250521-073156-77f1a2-pr6582.aar) |
| :heavy_check_mark: | 77f1a2e2 | [#15](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6582/15/) | 2025-05-21 07:35:12 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250521-073158-77f1a2-pr6582.zip) |
| :heavy_check_mark: | 77f1a2e2 | [#15](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6582/15/) | 2025-05-21 07:35:28 | ~3 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250521-073207-77f1a2-pr6582.zip) |
| :heavy_check_mark: | 77f1a2e2 | [#15](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6582/15/) | 2025-05-21 07:37:08 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250521-073159-77f1a2-pr6582.zip) |
| :heavy_check_mark: | 77f1a2e2 | [#15](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6582/15/) | 2025-05-21 07:37:11 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250521-073203-77f1a2-pr6582.zip) |
| :heavy_check_mark: | 77f1a2e2 | [#15](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6582/15/) | 2025-05-21 07:37:49 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250521-073204-77f1a2-pr6582.zip) |
| :heavy_check_mark: | 77f1a2e2 | [#15](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6582/15/) | 2025-05-21 07:43:22 | ~8 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6582/15//consoleText) |
| :heavy_multiplication_x: | 77f1a2e2 | [#15](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6582/15/) | 2025-05-21 08:32:43 | ~33 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6582/15//consoleText) |
| :heavy_check_mark: | 77f1a2e2 | [#16](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6582/16/) | 2025-05-21 09:11:42 | ~35 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6582/16//consoleText) |

shivekkhurana commented:
When we integrate Polygon, we'll need more changes as they implemented EIP-1559 slightly differently: https://github.com/ethers-io/ethers.js/discussions/2917

saledjenic committed:
fix(wallet)_: fee calculation improvements for status sepolia chain

Since the Status chain is gasless the old implementation  of `isEIP1559Compatible` function
(that was checking the base fee of `eth_feeHistory` in order to dynamically determine if the chain
is eip1559 compliant or not) was identifying it as a none eip1559 compliant chain, while it is for real.
These changes fix that by hardcodding the the flag for Status Sepolia chain.

saledjenic committed:
test_: flaky/failing tests fixed
